renormalizer/utils/configs.py

```diff
@@ -74,22 +86,26 @@ class CompressConfig:
     
     vrtol : float, optional
         The threshold of convergence in variational compression. Default is
-        :math:`10^-5`.
+        :math:`10^{-5}`.
     vguess_m : tuple(int), optional
         The bond dimension of ``compressed_mpo`` and ``compressed_mps`` to construct the initial guess 
         ``compressed_mpo @ compressed_mps`` in `MatrixProduct.variational_compress`.
         The default is (5,5).
-    
-    Note
-    ----
-    It is recommended to use the 2site algorithm in variational optimization,
-    though 1site algorithm is much cheaper. The reason is that 1site algorithm
-    is easy to stuck into local mimima while 2site algorithm is more robust. For
-    complicated Hamiltonian, the problem is severer.  The renormalized basis
-    selection rule used here to partly solve the local minimal problem is
-    according to Chan. J.  Chem. Phys. 2004, 120, 3172. For efficiency, you
-    could use 2site algorithm for several sweeps and then switch to 1site
-    algorithm to converge the final mps.
+    dump_matrix_size : int or float, optional
+        The total bytes threshold for dumping matrix into disks. Every local site matrix in the MPS
+        with a size larger than the specified size will be moved from memory to the disk at ``dump_matrix_dir``.
+        The matrix will be moved back to memory upon access.
```

Comment:
MPO also has this feature, right?

Reply:
Right. I run a few tests just now and it seems to be fine. The documentation could indeed be improved though. Since the title of the documentation is "MPS Compress Configuration".